config/assembleCoadd.py

```diff
@@ -2,10 +2,10 @@
 from lsst.obs.lsstSim.selectFluxMag0 import SelectLsstSimFluxMag0Task
 from lsst.pipe.tasks.scaleZeroPoint import SpatialScaleZeroPointTask
 
-config.select.retarget(SelectLsstImagesTask)
+#config.select.retarget(SelectLsstImagesTask)
```

Comment:
Why no longer retargeting this?  

I see selectLsstImages.py does seem a bit outdated (?). Some comments may be helpful.